test/functional/helpers/createFixture/clientScripts.js

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
Maybe add a bit more comments for the different functions making the library available and why it is being loaded that way.

Reply:
I added some more comments and shifted a couple things around. Not sure if that's any more helpful though. I tried!